### DISCUSSION TITLE
When i use pnpm@7.0.0-rc.3, The command pnpm init -y   failed

### DISCUSSION THREAD

**Discussion by myNameIsDu** [pnpm/pnpm#4556] [Q&A] [answered]
When i use pnpm@7.0.0-rc.3, The command `pnpm init -y`   failed

I don't understand
And i found a message about it
![image](https://user-images.githubusercontent.com/53305259/162885968-57b2ea9f-6597-4f51-8f22-8eea26a63940.png)
But it tells me that I don't have to add `--`, But now init I have to add it

**Comment by zkochan** [comment, marked as answer]
In v7, `pnpm init` is non-interactive by default, so just run `pnpm init` instead of `pnpm init -y`

**Reply by myNameIsDu** [reply]
I think it should be added to the version information, otherwise you don't know what's going on in this situation :- )

**Reply by myNameIsDu** [reply]
And it's still interactive 
<img width="534" alt="image" src="https://user-images.githubusercontent.com/53305259/163092483-010bfb2e-055c-4d14-bec0-b79dd6c14a38.png">
Is it because it's not stable version?
